app/main/arguments.js

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import yargs from 'yargs';
+
+export default yargs
+  .usage('Usage: $0 [options]')
+  .option('p', {
+    alias: 'profile',
```

Comment:
How could we make this match current behavior for usertesting packages?  i.e. in that case we want a unique profile, preferably matching the current ROOT behavior, and it won't be run via command line but by double clicking a binary